sap/5.9/modules/ROOT/pages/sap-connector-reference.adoc

```diff
@@ -262,15 +262,17 @@ SAP 5.6 and later supports updating a specific row in a table when a response is
 == Operations
 
 * <<asyncRfc>>
+* <<asynchronousRemoteFunctionCall>>
 * <<confirmTransactionId>>
 * <<getFunction>>
 * <<retrieveIdoc>>
 * <<send>>
 * <<startsaptransaction>>
 * <<syncRfc>>
+* <<synchronousRemoteFunctionCall>>
 
 [[asyncRfc]]
-=== Asynchronous Remote Function Call
+=== Asynchronous Remote Function Call (DEPRECATED)
```

Comment:
When deprecating, we should point to the new way of doing the same. Can we refer to the new operation `asynchronousRemoteFunctionCall`?